pom.xml

```diff
@@ -902,6 +902,71 @@
 
 			</properties>
 		</profile>
+		        
+		<!-- ========= [ HANDICAP INTERNATIONAL SPECIAL VERSION. ] ========= -->
+        <profile>
+			<id>sigmah4hi</id>
```

Comment:
Thanks @numero-six for your feedbacks ! Let's go for "custom" !
I've commited the changes which are now available in the pull request.


*--*Blog : http://www.grattecielhorizontal.info/
*  dernier post: *Idée d’un autre rapport de force face à la COP21
<http://www.grattecielhorizontal.info/Idee-d-un-autre-rapport-de-force-face-a-la-COP21>

2016-02-02 10:35 GMT+01:00 Brendan Le Ny <notifications@github.com>:

> In pom.xml
> <https://github.com/sigmah-dev/sigmah/pull/4#discussion_r51543675>:
>
> > @@ -902,6 +902,71 @@
> >
> >  			</properties>
> >  		</profile>
> > +		
> > +		<!-- ========= [ HANDICAP INTERNATIONAL SPECIAL VERSION. ] ========= -->
> > +        <profile>
> > +			<id>sigmah4hi</id>
>
> @osarrat <https://github.com/osarrat> yes you're right, NGO is not an
> option. I'm not ok with userbuild because user may refer to pretty
> anybody. I'm ashamed I didn't thought about custom before since it's seem
> quite obvious.
>
> Is "sigmah-custom-*" ok for you?
>
> You just have to commit on your local repo (master branch) and push it to
> your own github fork. The current pull-request should consider your change.
> You may also fix the indentation which seem broken (line 907).
>
> I'm sorry this discussion takes time but naming things and making things
> unambiguous for everyone, including new developers, is very important.
> Also, this is a nightmare to changer later so i prefer take everyone two
> minutes now than having support to do later.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/sigmah-dev/sigmah/pull/4/files#r51543675>.
>